source/_integrations/nfandroidtv.markdown

```diff
@@ -125,4 +125,20 @@ icon:
   auth: "digest"
 ```
 
+Example of automation serivce call, full cofiguration:
+
+```yaml
+  - service: notify.living_room_tv
+    data:
+      title: "Thanks for the water!"
+      message: "Nigel is {{ states.sensor.nigel_moisture.state }}% moisture"
```

Comment:
Don't access `states` directly, as it won't be able to handle occurrences when the entity isn't available/loaded yet.

Instead use the `states()` function

Reply:
Understood.  Not quite sure how to edit it but I will try or resubmit a PR if it does not work.  (Still learning) :-)

Reply:
Hi Frenck.  Big fan of your work.   I'm a bit new to using git etc and appreciate your patience.  
I agree with the changes/suggestions. I'm not sure how to implment the states() change - I have it fixed but unsure if I need cancel this PR (unlikely) or somehow edit this one?  I'll do some git research to address it.  thx.